src/middlewared/middlewared/plugins/enclosure_/slot_mappings.py

```diff
@@ -121,11 +155,35 @@ def get_slot_info(model, enclosures=None):
                     'id': {
                         ses_logical_ids[min_id]: {
                             # 1 - 24
-                            i: {'orig_slot': i, 'mapped_slot': i} for i in range(1, 25)
+                            i: {'sysfs_slot': i, 'mapped_slot': i} for i in range(1, 25)
```

Comment:
I think strings `sysfs_slot` and `mapped_slot` should be defined as constants, e.g. `SYSFS_SLOT_KEY` and `MAPPED_SLOT_KEY` just because they're used so many times it will be easy to gloss over a typo in this file.

Reply:
Good point, fixed.